client/components/PubEdge/pubEdge.scss

```diff
@@ -0,0 +1,78 @@
+@import 'styles/variables.scss';
+@import 'containers/Pub/PubDocument/pubBody.scss';
+
+.pub-edge-component {
+	@include pub-body-styles();
```

Comment:
I'm curious how much you're getting out of this mixin — if it's only a few selectors or properties, I'd encourage you to either duplicate them or break out the functionality you need into a smaller mixin. We can save a few bytes, and make these styles easier to reason about — this way.